crates/bevy_render/src/color/mod.rs

```diff
@@ -392,7 +392,7 @@ impl Color {
         }
     }
 
-    /// Set red in sRGB colorspace.
+    /// Set red in sRGB colorspace, returned value is of variant [`Color::Rgba`].
```

Comment:
```suggestion
    /// Set red in sRGB colorspace, `self` is modified to variant [`Color::Rgba`] and returned.
```
And same for this other `set_*` methods, or a better variant. Mentioning only the returned value is changed to the variant could be understood as self is not changed too